sql/_23_apricot_qa/_02_performance/_02_function_based_index/answers/function_based_index_coverage_003.answer

```diff
@@ -1,28 +1,56 @@
 ===================================================
 0
 ===================================================
-0
+Error:-494
```

Comment:
Why did the results change?

Reply:
During the modification of test cases, the same table name has existed.
So I added a 'drop table' in the test case to prevent errors.